Mentor tools - Spanish Translations (#4)
ropensci-org/champions-mentor-guidelines: mentors-tools.es.qmd

> ## Herramientas para tutores

Author: ```suggestion
## Herramientas para mentoras/es
```

### Infraestructura {#infrastructure}

> Configuramos una serie de herramientas para ayudarte en tu tutoría.

Author: ```suggestion
Configuramos una serie de herramientas para ayudarte en tu mentoría.
```

- **Google Drive:** la unidad contiene:
  

>   - Una carpeta para cada campeón. Tienes acceso a la carpeta de tu pupilo

Author: ```suggestion
  - Una carpeta para cada campeón/a. Tienes acceso a la carpeta la persona que mentoreas.
```

>     solamente. Esta carpeta tiene ahora un documento con la solicitud de tu mentee. En el futuro también subiremos vídeos u otro material que tu mentee genere como parte de su trabajo en el programa.  También puedes utilizarla para almacenar y compartir recursos con tu mentee.

Author: ```suggestion
 Esta carpeta tiene ahora un documento con la aplicación completa. En el futuro también subiremos vídeos u otro material que se generen como parte de su trabajo en el programa.  También puedes utilizarla para almacenar y compartir recursos.
```

>   - Una carpeta común para los mentores con:

Author: ```suggestion
  - Una carpeta común para mentores-as con:
```

    
    - Las diapositivas de la reunión inicial.
    
    - El enlace a este libro.
    

>     - Las diapositivas de la formación de mentores.

Author: ```suggestion
    - Las diapositivas de la formación de mentores-as.
```

    
    - Las plantillas.
    

>     - Material de marca rOpenSci.

Author: ```suggestion
    - Material de la marca de rOpenSci (como logos, plantillas de slides, etc).
```

> - **Plantillas:** Hemos creado plantillas para la reunión con tu alumno.

Author: ```suggestion
- **Plantillas:** Hemos creado plantillas para las reuniones con tu campeon/a
```

  Hay 3 plantillas diferentes: una para la primera reunión, otra para

>   la última reunión y otra para el resto de la reunión. En [*plantillas

Author: ```suggestion
  la última reunión y otra para el resto de las reuniones. En el [documento de google de *plantillas](https://docs.google.com/document/d/1VtNYfDNYRS7xAbs1Ajf2S5wsv3mOQ0BcTn93cXAv6sc/)
```

>   google doc*](https://docs.google.com/document/u/0/d/1vjmfbjv9ABJ6fkbtQRMFPydJZ3ypKOIpZ-xf7_iBhvo/edit)

Author: ```suggestion
  
```

>   se puede copiar y modificar.

Author: ```suggestion
  podes econtrar estas sugerencias de agenda.  El documento se puede copiar, adaptar y modificar para su uso.
```

- **Slack:** tenemos tres canales privados para tu comunicación con

>   otros mentores, con los campeones y con los participantes de las cohortes previas:

Author: ```suggestion
  las personas que participan de esta cohorte y con los participantes de las cohortes previas:
```

>   - **\#mentores-2024**: sólo mentores en este canal. Puedes preguntar

Author: ```suggestion
  - **\#mentors-2025**: sólo mentores/as en este canal. Puedes hacer
```

>     preguntas, compartir recursos, pedir consejo a tus compañeros. Yani está en este espacio para

Author: ```suggestion
    preguntas, compartir recursos, pedir consejo a tus compañeros/as. Yani está en este espacio para
```

>     proporcionar apoyo y compartir información para los mentores.

Author: ```suggestion
    proporcionar apoyo y compartir información relevante.
```

>   - **\#campeones-y-mentores-2024:** un canal con todos los participantes Yani

Author: ```suggestion
  - **\#champions-and-mentors-2025:** un canal con todos los participantes. Yani
```

    y Maëlle están en este canal para prestar apoyo y compartir
    información común.
  

>   - ** programa #campeones:** un canal con todos los participantes del programa. Yani

Author: ```suggestion
  - ** champions-program:** un canal con todos los participantes del programa. Yani
```

    y Maëlle están en este canal para ofrecer apoyo y compartir
    información común.

### Herramientas para las reuniones {#meeting-tools}

Todas nuestras reuniones en línea utilizarán

- Software de videoconferencia

- Notas colaborativas

- Herramientas de programación

- Código de conducta

> La siguiente lista es un resumen de las herramientas que utilizaremos en nuestro

Author: ```suggestion
La siguiente lista es un resumen de las herramientas que utilizaremos en nuestras
```

sesión de cohorte completa (formación y reuniones) junto con las alternativas que

> puedes utilizar en tus reuniones de tutoría. Si estás acostumbrado a utilizar una herramienta

Author: ```suggestion
puedes utilizar en tus reuniones de tutoría. Si tienes la costumbre de usar una herramienta
```

que no está en esta lista, puedes utilizarla (y hacernos saber cuál es
para que podamos añadirla a la lista).

#### Videoconferencia {#videocalls}

> Para nuestras Llamadas de Cohorte utilizaremos Zoom. Enviaremos un calendario con las

Author: ```suggestion
Para nuestras Llamadas de Cohorte utilizaremos Zoom. Enviaremos un calendario con el
```

horario completo del programa y los enlaces de zoom y la reunión compartida
a tu correo electrónico. Puedes añadir este calendario a tu agenda.

> Para las reuniones de tutoría, normalmente llamadas 1:1 con tu tutelado, eres libre

Author: ```suggestion
Para las reuniones de mentoría, normalmente llamadas 1:1 con tu campeon/a, eres libre
```

> de utilizar cualquier herramienta que ambos acordéis. Aquí tienes una lista de servicios:

Author: ```suggestion
de utilizar cualquier herramienta que acuerden. Aquí tienes una lista de servicios:
```

- [*Zoom*](http://zoom.us/): también puedes utilizar tus propias salas Zoom. En
  cuenta gratuita permite 40 minutos de llamada para reuniones 1 a 1.

- [*Slack*](https://slack.com/help/articles/216771908-Make-calls-in-Slack):
  Slack permite llamadas entre los miembros del espacio de trabajo. Puedes utilizar

>   esta funcionalidad en rOpenSci Slack.

Author: ```suggestion
  esta funcionalidad en el Slack de rOpenSci.
```

- [Google Meet](https://meet.google.com/landing): si utilizas google calendar para programar tu

>   llamadas de tutoría, añadir una llamada es sencillo, haz clic en el botón

Author: ```suggestion
  llamadas de mentoría, puedes añadir una llamada haciendo clic en el botón
```

  'Añadir videoconferencia de Google Meet'.

- [*Jitsi:*](http://meet.jit.si/) una alternativa de código abierto. Integra
  con Google Calendar y Slack.

> - [*Skype*](http://skype.com/): funciona bien en zonas con menor ancho de banda.

Author: ```suggestion

```

> - [*WhatsApp*](https://www.whatsapp.com/): funciona bien para los bajos

Author: ```suggestion
- [*WhatsApp*](https://www.whatsapp.com/): funciona bien en zonas con menor 
```

>   de ancho de banda.

Author: ```suggestion
  ancho de banda.
```

> - [*Telegrama*](https://telegram.org/): también es genial para un ancho de banda bajo

Author: ```suggestion
- [*Telegrama*](https://telegram.org/): también es genial para zonas con un ancho de banda bajo.
```

>   zonas.

Author: ```suggestion
  
```

> - [*Por lo que*](https://whereby.com/) tiempo ilimitado para el cara a cara

Author: ```suggestion
- [*Whereby*](https://whereby.com/) tiempo ilimitado para reuniones uno a uno.
```

>   reuniones.

Author: ```suggestion

```

> ### Notas de colaboración {#note-taking}

Author: ```suggestion
### Notas colaborativas {#note-taking}
```

> Utilizaremos Google Docs para nuestras llamadas de Cohorte. Para tu tutoría 1:1

Author: ```suggestion
Utilizaremos Google Docs o una solucion simil Etherpad (si es requedida por accesibilidad) para nuestras llamadas de Cohorte. Para tu mentoría 1:1
```

puedes utilizar cualquier herramienta de notas colaborativas.

> - [Google Docs](https://docs.google.com/) Para utilizar Google Docs en tus reuniones de tutoría,

Author: ```suggestion
- [Google Docs](https://docs.google.com/) Para utilizar Google Docs en tus reuniones de mentoría,
```

  crea un documento en tu carpeta de Google Drive.
  Puedes hacer una copia de la plantilla que te proporcionamos.

- [Hack.md:](https://hackmd.io/) Esta es una opción accesible y tienen
  formato Markdown para el texto.
  Se puede integrar con GitHub.

- [Formato Etherpad](https://pad.riseup.net/) esta es otra opción accesible con formato Markdown para el texto y no necesita una cuenta de usuario.
  El inconveniente es que sólo está disponible durante un tiempo (puedes configurar cuántos días de una lista de opciones).

> ### Herramientas de programación {#scheduling}

Author: ```suggestion
### Herramientas para organizar y agendar reuniones {#scheduling}
```

Utilizamos herramientas para acordar el día y la hora en que nos reuniremos.

> Quizá para ti sea más fácil acordarlo por correo electrónico,

Author: ```suggestion
Quizás la forma más fácil acordar las reuniones sea por correo electrónico,
```

pero aquí tienes una lista de herramientas que pueden ayudarte en esta tarea (especialmente porque las zonas horarias pueden ser muy complicadas).

- [*Calendly*](https://calendly.com/): puedes enlazar tu calendario con
  Calendly y proporcionar un enlace a otras personas para que elijan un día y una hora
  en tu agenda. Es gratuito sólo para un tipo de evento. También

>   ofrece una opción de billar. Tiene un límite de 40 plazas y es pantalla

Author: ```suggestion
  ofrece una opción de encusta Tiene un límite de 40 lugares y es accesible para lectores de pantalla.
```

  lector de pantalla.

> - [*Garabato:*](https://doodle.com/) la versión gratuita muestra anuncios. Es

Author: ```suggestion
- [*Doodle:*](https://doodle.com/) la versión gratuita muestra anuncios. Es
```

  accesible para lectores de pantalla. Los anuncios son muy molestos para la navegación de la herramienta.

>   Tú eliges qué día y a qué horas puedes utilizarla,

Author: ```suggestion
  Tú eliges qué día y a qué horas de esos dias estas disponible,
```

>   y ofrece una tercera opción, que es "Tal vez".

Author: ```suggestion
  y ofrece una tercera opción, que es "Tal vez" y que permite organizar con una mayor flexibilidad que cuando las opcopnes son "Si Puedo" y "No puedo".
```

  Es accesible para lectores de pantalla.

> - [*Cuándo2meet:*](https://www.when2meet.com/) es gratuito, te permite

Author: ```suggestion
- [*When2meet:*](https://www.when2meet.com/) es gratuito, te permite
```

  elegir un día y un intervalo de tiempo y crear un enlace para llenar la reserva.
  Puedes decir sí o no. No es accesible para lectores de pantalla.

> - Google Calendar: si tienes un plan de empresa, puedes ofrecer plazas en

Author: ```suggestion
- Google Calendar: si tienes un plan de empresa, puedes ofrecer lugares libres en
```

>   tu agenda para que la gente reserve contigo.

Author: ```suggestion
  tu agenda para que la gente reserve reuniones contigo.
```

- [*Hora y fecha*](http://timeanddate.com/): para comprobar los husos horarios
  y calcular fechas y horas.

- [*Compañero de la Hora Mundial*](https://www.worldtimebuddy.com): para comprobar los husos horarios
  y calcular fechas y horas.

### Código de conducta

rOpenSci se compromete a proporcionar un ambiente seguro, inclusivo, acogedor y

> y libre de acoso para todos. Tenemos una [*Código de

Author: ```suggestion
y libre de acoso para todas las personas. Tenemos un [*Código de
```

> Conducta*](https://ropensci.org/code-of-conduct/) que incluye el

Author: ```suggestion
Conducta*](https://ropensci.org/code-of-conduct/) que incluye instrucciones
```

> directrices de información para reforzar nuestro CdC. Alternativamente, puedes

Author: ```suggestion
pare reportar incidentes y como es el proceso para aplicar nuestro CdC. Alternativamente, puedes
```

enviar un correo electrónico a '[conduct@ropensci.org](mailto:conduct@ropensci.org) que se enviará a los miembros del Comité.

### Divulgación y contribuciones

> En la segunda parte del año, los campeones planificarán y llevarán a cabo actividades de divulgación en sus comunidades locales.

Author: ```suggestion
En la segunda parte del año, campeones/as planificarán y llevarán a cabo actividades de divulgación, de preferencia en sus comunidades locales, pero también en comunidades internacionales.
```

> Estas actividades pueden ser talleres, charlas, participación en actos y conferencias, publicaciones en blogs o entrevistas.  Pueden organizarse con instituciones, comunidades, grupos de usuarios y organizaciones locales y regionales; de hecho, ¡es muy deseable que se organicen así!

Author: ```suggestion
Estas actividades pueden ser talleres, charlas, participación en eventos y conferencias, publicaciones en blogs o entrevistas.  Pueden organizarse con instituciones, comunidades, grupos de usuarios y organizaciones locales y regionales; de hecho, ¡es muy deseable que se organicen así!
```

> Por supuesto, estas actividades pueden estar dentro del abanico de posibilidades de contribución que ya tiene rOpenSci, como las Convocatorias Comunitarias, la sesión de co-working, la entrada en el blog de rOpenSci y las traducciones.

Author: ```suggestion
Por supuesto, estas actividades pueden estar dentro del abanico de posibilidades de contribución que ya tiene rOpenSci, como las "Conversaciones con la comunidad", las sesiones de co-working, los mini-hackatones, articulos en el blog de rOpenSci y las traducciones.
```

Estos recursos pueden ayudarte a ver qué tipo de contribuciones y participaciones se pueden hacer en rOpenSci y con estas ideas planificar la actividad de difusión del programa:

> - [Hoja de consejos](https://ropensci.org/blog/2022/09/13/contributing-ropensci/) Esta hoja de consejos contiene un resumen de las distintas formas de contribuir y participar en rOpenSci.

Author: ```suggestion
- [Hoja de consejos](https://ropensci.org/es/blog/2022/09/13/contributing-ropensci-es/) Esta hoja de consejos contiene un resumen de las distintas formas de contribuir y participar en rOpenSci.
```

> - [Guía de contribución](https://contributing.ropensci.org/) tiene más detalles, enlaces, materiales y ejemplos de cada forma de participar en nuestra comunidad.

Author: ```suggestion
- [Guía de contribución](https://contributing.ropensci.org/) tiene más detalles, enlaces, materiales y ejemplos de cada forma de participar en nuestra comunidad. (En ingles).
```

> - [Guía del blog](https://blogguide.ropensci.org/): este libro explica cómo contribuir con una entrada de blog para el sitio web de rOpenSci.

Author: ```suggestion
- [Guía del blog](https://blogguide.ropensci.org/): este libro explica cómo escribir un blog post para el sitio web de rOpenSci (en Ingles).
```
